packages/synapse-react-client/src/components/CreateTableViewWizard/CreateTableViewWizard.integration.test.tsx

```diff
@@ -0,0 +1,793 @@
+    expect(nextButton).toBeDisabled()
+
+    // Add an item to scope
+    const addContainersToScopeButton = await screen.findByRole('button', {
+      name: 'Add Containers',
+    })
+    await userEvent.click(addContainersToScopeButton)
+    await screen.findByTestId('EntityFinderModalMocked')
+    const entityFinderModalPassedProps = mockEntityFinderModal.mock.lastCall![0]
+    act(() => {
+      entityFinderModalPassedProps.onConfirm([{ targetId: 'syn123' }])
+    })
+    // The scope and mask are now valid -- Next should no longer be disabled
+    await waitFor(() => expect(nextButton).not.toBeDisabled())
+
+    // Uncheck all mask boxes to make the mask invalid
+    await userEvent.click(screen.getByLabelText('Files'))
+
+    // Verify that we cannot advance
+    await waitFor(() => expect(nextButton).toBeDisabled())
+  })
+
+  it('Shows an error when column model creation fails', async () => {
+    const errorMessage = 'Mocked error in POST /column/batch'
+    server.use(
+      rest.post(
+        `${getEndpoint(
+          BackendDestinationEnum.REPO_ENDPOINT,
+        )}/repo/v1/column/batch`,
+        async (req, res, ctx) => {
+          return res(
+            ctx.status(400),
+            ctx.json({
+              reason: errorMessage,
+            }),
+          )
+        },
+      ),
+    )
+
+    const onComplete = jest.fn()
+    const onCancel = jest.fn()
+    renderComponent({
+      open: true,
+      parentId: mockProjectEntityData.id,
+      onComplete,
+      onCancel,
+    })
+
+    // Select "Table"
+    const tableButton = await screen.findByRole('menuitem', {
+      name: 'Table',
+    })
+    await userEvent.click(tableButton)
+
+    // Add a few columns
+    await addColumnModelToForm('foo')
+    await addColumnModelToForm('bar')
+
+    await userEvent.click(screen.getByRole('button', { name: 'Next' }))
+
+    // Give the table a name
+    const nameField = await screen.findByRole('textbox', { name: 'Name' })
+    await userEvent.type(nameField, 'tableName')
+    await userEvent.click(screen.getByRole('button', { name: 'Finish' }))
+
+    // Verify that the error is shown and the callback was not invoked
+    await waitFor(() => {
+      expect(createColumnModelsSpy).toHaveBeenCalledWith(
+        MOCK_ACCESS_TOKEN,
+        expect.arrayContaining([
+          {
+            name: 'foo',
+            columnType: 'STRING',
+            maximumSize: 50,
+          },
+          {
+            name: 'bar',
+            columnType: 'STRING',
+            maximumSize: 50,
+          },
+        ]),
+      )
+
+      // The error should be shown
+      const alert = screen.getByRole('alert')
+      within(alert).getByText(errorMessage)
+
+      // The call to create the entity and the onComplete callback should not have been called
+      expect(createEntitySpy).not.toHaveBeenCalled()
+      expect(onComplete).not.toHaveBeenCalled()
+    })
+  })
+
+  it('Shows an error when entity creation fails', async () => {
```

Comment:
Could update one of these tests to demonstrate that the error is not shown on previous screens, if not applicable